jenkins-scripts/lib/project-install-homebrew.bash

```diff
@@ -73,10 +73,10 @@ echo '# END SECTION'
 
 echo "#BEGIN SECTION: brew doctor analysis"
 brew missing || brew install $(brew missing | awk '{print $2}') && brew missing
-# if sdl installed, skip brew doctor
-# remove this line when open-scene-graph stops depending on sdl
-# https://github.com/Homebrew/homebrew-core/pull/81101
-brew list | grep '^sdl$' || brew doctor
+# if proj@7 installed, skip brew doctor
+# remove this line when gdal stops depending on a deprecated version of proj
+# https://github.com/Homebrew/homebrew-core/issues/82441
+brew list | grep '^proj@7$' || brew doctor || echo MARK_AS_UNSTABLE
```

Comment:
Since `brew doctor` is now a soft failure rather than a hard one does it make sense to run it unconditionally and live with with the warning or otherwise let it warn about other issues while suppressing the proj complaint?

Reply:
I'd prefer to suppress the `proj@7` complaint since we don't have any control over that; that's an issue with homebrew-core. `brew doctor` usually complains about out-of-date Xcode version on our build machines, which I like to be notified about.